### PR TITLE
UI: sliding subviews for trackers blocked: this page, all time

### PR DESCRIPTION
**Pull request by laurengarcia**
**Reviewer:**
@russellholt @jdorweiler 

## Description:
https://app.asana.com/0/263565713128358/362146958886747

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

**Comment by jdorweiler** [comment]
I saw the page list was cut off sometimes.
screen shot 2017-06-26 at 8 28 14 am

**Comment by laurengarcia** [comment]
Overflow fixed now.

File: js/models/trackerlist-site.es6.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                                  name: companyName,
+                                  count: companyName === 'unknown' ? 0 : company.count,
+                                  px: Math.floor(company.count * 228 / maxCount),
+                                  urls: company.urls
+                              }
+                          })
+                          .sort((a, b) => {
+                              return a.count - b.count;
+                          })
+                          .reverse();
```
Comment (jdorweiler) [inline js/models/trackerlist-site.es6.js]: Could you drop the `reverse` and do `b.count - a.count` above?

**Comment by laurengarcia** [comment]
K, turns out @jdorweiler didn't have all commits from this branch, sort issue on page list was fixed on Fri and bug seems to be resolved.

File: js/models/trackerlist-site.es6.js (hunk 1 above)
Comment (laurengarcia) [inline js/models/trackerlist-site.es6.js]: Sure, just committed and sent pushed change to origin ~10 mins ago, but github doesn't seem to be displaying the commit even tho my cli says it went thru and is up-to-date... ?! Maybe there's some issue with github display but commit appears to have arrived here, never seen an issue like this before but i heard they had an outtage last week(?):
screen shot 2017-06-26 at 6 28 07 pm

Comment (laurengarcia) [inline js/models/trackerlist-site.es6.js]: Ok, I created a whitespace commit and pushed to origin, now branch has sort commit but not latest whitespace commit, but whitespace commit doesn't matter. GitHub having issues today. But yeah, this is fixed and now appearing as recent commit in PR now.

Comment (jdorweiler) [inline js/models/trackerlist-site.es6.js]: Ok cool. I was having that same problem last week too.

**Pull request merged**